src/Core/src/Core/IEntry.cs

```diff
@@ -4,5 +4,6 @@ public interface IEntry : IView, IText
 	{
 		bool IsPassword { get; }
 		bool IsTextPredictionEnabled { get; }
+		bool IsReadOnly { get; }
```

Comment:
Could we move this to `ITextInput`?

Reply:
We can, and it was but @mattleibow suggested that we use it as a property when we figure out what to do with other properties in ITextInput, please @jsuarezruiz check the previous comments :)

Reply:
I am just removing everything that is not implemented yet from ITextInput. It should not have gone in it it was not implemented yet.

Reply:
@jsuarezruiz and @mattleibow can we review it now? please 😄